lib/edit-site-page.php

```diff
@@ -36,6 +36,55 @@ function gutenberg_is_edit_site_page( $page ) {
 	return in_array( $page, $allowed_pages, true );
 }
 
+/**
+ * Load editor styles (this is copied form edit-form-blocks.php).
+ * Ideally the code is extracted into a reusable function.
+ *
+ * @return array Editor Styles Setting.
+ */
+function gutenberg_get_editor_styles() {
+	global $editor_styles;
+
+	//
+	// Ideally the code is extracted into a reusable function.
+	$styles = array(
+		array(
+			'css' => file_get_contents(
+				ABSPATH . WPINC . '/css/dist/editor/editor-styles.css'
+			),
+		),
+	);
+
+	/* translators: Use this to specify the CSS font family for the default font. */
+	$locale_font_family = esc_html_x( 'Noto Serif', 'CSS Font Family for Editor Font' );
```

Comment:
There is an [unhappy test](https://travis-ci.com/github/WordPress/gutenberg/jobs/299368176) with this line of code, this should fix it:

```suggestion
	$locale_font_family = esc_html_x( 'Noto Serif', 'CSS Font Family for Editor Font', 'default' );
```

Reply:
I think it should be "gutenberg" if we follow the other cases where we have translation on the server.

Reply:
Just found one other case (`esc_html__` in template-loader.php). Changed it to `gutenberg`.